javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/FixedSchemaFactory.java

```diff
@@ -39,6 +39,7 @@
     public static final String SNAPSHOT_COMMIT_FK =    "commit_fk";
     public static final String SNAPSHOT_GLOBAL_ID_FK = "global_id_fk";
     public static final String SNAPSHOT_TYPE =         "type";
+    public static final String VERSION =               "version";
```

Comment:
@milanov there are few bugs here:
- NPE is thrown when version field is missing in a snapshot stored in SQL or Mongo (created by javers <= 1.4.2)
migration script is optional, so JaVers cant crash when someone doesnt run it
(I will fix it)
- version is ALWAYS deserialized to 1
take a look at line 60 in CdoSnapshotTypeAdapter. When snapshots are deserialized from DB, previous field is null, 
and builder overwrites number set in `.withVersion(version)`
- version field is never write to Sql database, you simply forgot to add it to `CdoSnapshotRepository.insertSnapshot()`